File: FrontEnd/ClientApp/src/app/models/ResponseTarifa.ts
import { Planos } from "./Planos";

export class ResponseTarifa {
    origem: number;
    destino: number;
    precoMinuto: number;
    tempo: number;
    plano: Planos;
    comFaleMais: number;
    somFaleMais: number;
}

File: FaleMais.Domain/DTO/ResponseTarifa.cs
using FaleMais.Domain.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace FaleMais.Domain.DTO
{
    public class ResponseTarifa : Models.Tarifa
    {
        public int Tempo { get; set; }
        public Planos? Plano { get; set; }
        public double ComFaleMais { get; set; }
        public double SemFaleMais { get; set; }
    }
}
